packages/frontend/src/constans/Fonts.tsx

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
nadal bym jakas nazwe. Bo jak zapiszesz tak to obawiam sie ze nie zadziała autoimport